moveit_msgs/RobotTrajectory trajectoryToTransform
---
moveit_msgs/RobotTrajectory transformedTrajectory
